client/src/store.js.orig

```diff
@@ -0,0 +1,12 @@
+const store = {
```

Comment:
looks like this file appeared because of a Git issue, it should be safe to delete